src/CacheVariables.jl

```diff
@@ -12,26 +13,22 @@ function _cachevars(ex::Expr)
     return Vector{Symbol}(undef,0)
 end
 
-macro cache(path,ex::Expr,overwrite=false)
+macro cache(path, ex::Expr, overwrite::Bool=false)
     vars = _cachevars(ex)
-    vardesc = join(string.(vars),", "," and ")
-    if !isfile(path) || overwrite
-        varkws  = [:($(var) = $(var)) for var in vars]
-        varlist = :($(varkws...),)
-        return quote
-            val = $(esc(ex))
-            print("Saving variables ",$vardesc," to cache file `",$path,"`...")
-            bson($path; $(esc(varlist))...)
-            println("done!")
-            val
-        end
-    else
-        vartuple = :($(vars...),)
-        return quote
-            print("Loading variables ",$vardesc," from cache file `",$path,"`...")
-            data = BSON.load($path)
-            $(esc(vartuple)) = getindex.(Ref(data),$vars)
-            println("done!")
+    vardesc = join(string.(vars), "\n")
+    varkws  = [:($(var) = $(var)) for var in vars]
+    varlist = :($(varkws...),)
+    vartuple = :($(vars...),)
+
+    return quote
+        if !isfile($(esc(path))) || $(esc(overwrite))
```

Comment:
```suggestion
        if !isfile($(esc(path))) || $overwrite
```

I think `overwrite` probably does not need to be escaped, but would need to experiment a bit to be sure.